ccl/workloadccl/allccl: TestAllRegisteredWorkloadsValidate/tpcc failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/26a06a661e7c1ce63f3077269005472182286ca4

Parameters:
```
TAGS=
GOFLAGS=-race
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=596131&tab=buildLog

```


ERROR: signal: killed

1 runs completed, 1 failures, over 7m23s
context canceled
make: *** [stress] Error 1
Makefile:848: recipe for target 'stress' failed
```

--- a/pkg/ccl/workloadccl/allccl/all_test.go
+++ b/pkg/ccl/workloadccl/allccl/all_test.go
@@ -19,6 +19,7 @@ import (
 	_ "github.com/cockroachdb/cockroach/pkg/ccl"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
+	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/workload"
 )
@@ -54,6 +55,10 @@ func TestAllRegisteredWorkloadsValidate(t *testing.T) {
 		}
 
 		t.Run(meta.Name, func(t *testing.T) {
+			if meta.Name == `tpcc` && (testing.Short() || util.RaceEnabled) {
+				t.Skip(`tpcc loads a lot of data`)
+			}
+
 			ctx := context.Background()
 			s, db, _ := serverutils.StartServer(t, base.TestServerArgs{
 				UseDatabase: "d",

